bucket/tic-80.json

```
@@ -1,10 +1,10 @@
 {
-    "homepage": "https://tic.computer/",
-    "description": "Fantasy computer for making, playing and sharing tiny games",
-    "version": "0.70.6",
+    "homepage": "https://tic80.com",
+    "description": "TIC-80 is a fantasy computer for making, playing and sharing tiny games",
+    "version": "0.80.1344",
     "license": "MIT",
-    "url": "https://github.com/nesbox/TIC-80/releases/download/v0.70.6/tic80_0.70.6.zip",
-    "hash": "8c1aeeac4370142609a151f39577df7b0c3f2dfe6e1592ef351c4fb85dce4cb0",
+    "url": "https://github.com/nesbox/TIC-80/releases/download/v0.80.1344/tic80-v0.80-win.zip",
+    "hash": "0C243F63F56ECF2822B876A846CBACEBD4E3D5ACACA12F35CC1AF4F8DF32F9F9",
```

Hashes are stored in lowercase for consistency:

```suggestion
    "hash": "0c243f63f56ecf2822b876a846cbacebd4e3d5acaca12f35cc1af4f8df32f9f9",
```